stl/inc/chrono

```diff
@@ -2038,9 +2069,9 @@ namespace chrono {
         };
 
         // __std_tzdb_get_leap_seconds gets leap second (LS) data from the registry, but only if it contains more
-        // LSs than we already know about. The registry only contains LSs after 2018, so we need to tell it how many of
-        // *those* we already know about. The *total* number of LSs known at this point is a combination of what the
-        // caller knows (_Current_size, 0 on first call) and the _Known_leap_seconds entries.
+        // LSs than we already know about. The registry only contains LSs after 2018, so we need to tell it how many
+        // of *those* we already know about. The *total* number of LSs known at this point is a combination of what
+        // the caller knows (_Current_size, 0 on first call) and the _Known_leap_seconds entries.

```

Comment:
Yeah, I missed this because the damage got done while interactivly resolving rebase comments after having reviewed things. I should ensure I review things a final time and perhaps turn off "format on save" when resolving such conflicts.